src/constants/countries.js

```diff
@@ -0,0 +1,22 @@
+export const countries = [
+  {
+    name: 'United States',
+    code: 'US'
+  },
+  {
+    name: 'United Kingdom',
+    code: 'GB'
+  },
+  {
+    name: 'France',
+    code: 'FR'
+  },
+  {
+    name: 'Australia',
+    code: 'AU'
+  },
+  {
+    name: 'The Netherlands',
+    code: 'NL'
+  },
```

Comment:
Could you add Poland here? I want to test it because in most cases there is no data for Poland.

Reply:
Sure, added. :)